.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -12,10 +12,6 @@ jobs:
         CONFIG: linux_64_numpy1.22python3.10.____cpythonpython_implcpython
         UPLOAD_PACKAGES: 'True'
         DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cos7-x86_64
-      linux_64_numpy1.22python3.9.____73_pypypython_implpypy:
-        CONFIG: linux_64_numpy1.22python3.9.____73_pypypython_implpypy
-        UPLOAD_PACKAGES: 'True'
-        DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cos7-x86_64
```

Comment:
We shouldn't be dropping the pypy (or py312) builds for 1.x. AFAICT this happens because the migrator for those can only add _one_ key, not two (after the duplication here). More details in https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/5790.